department.rb

```diff
@@ -0,0 +1,39 @@
+class Department
+
+  attr_reader :name
+
+  def initialize( name )
```

Comment:
In Ruby, most folks are going to not have spaces between the parens and the parameter names.  In other words, the convention is to write your method definitions like this:

`def initialize(name)`

Very minor, though.